onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5132,10 +5132,14 @@ def aten_sspaddmm(
     raise NotImplementedError()
 
 
-def aten_stack(tensors: Sequence[TensorType], dim: int = 0) -> TensorType:
+@torch_op("aten::stack")
+def aten_stack(
+    tensors: Union[TTensorOrString, Sequence[TTensorOrString]], dim: int = 0
```

Comment:
What does it do if the input is a tensor, instead of a sequence? Does it act like an identity op then?

Inference for `ConcatFromSequence` will fail if the input is not a sequence, I think.

Reply:
Ahh Yes, there might be a chance it's a Tensor input for this ATen op, I should either change the annotation, or add an TypeError.